apps/server/src/modules/copy-helper/types/copy.types.ts

```diff
@@ -6,12 +8,13 @@ export type CopyStatus = {
 	type: CopyElementType;
 	status: CopyStatusEnum;
 	elements?: CopyStatus[];
-	copyEntity?: BaseEntity;
-	originalEntity?: BaseEntity;
+	copyEntity?: BaseEntity | AuthorizableObject;
```

Comment:
Wouldn't be `AuthorizableObject` enough here because we're only interested in the `id` property?

Reply:
good point, BaseEntity actually implements AuthorizableObject